server/services/star.js

```diff
@@ -254,7 +276,15 @@ module.exports = class StarService extends EventEmitter {
     }
 
     calculateTerraformedResources(naturalResources, terraforming) {
-        return (terraforming * 5) + naturalResources;
+        return {
```

Comment:
Move to a `starResourceService`?

Reply:
I think we want to floor these results.

Reply:
Would it be easier to keep the original function and use it like this:

`calculateTerraformedResources(star.naturalResources.economy, terraforming)`, this ensures that we only calculate what is actually required. No point in calculating science if we only need eco.